test_corpora_dictionary_2574.py

```diff
@@ -0,0 +1,375 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
```

Comment:
@piskvorky What should we do about the license header in contributed files?

Reply:
Keep our standard header, everywhere.

And if a contributor doesn't agree with the copyright assignment, we cannot accept the contribution (too much headache).

Reply:
Sorry about the license header mix-up. Ill close this PR.